go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -94,8 +94,8 @@ var (
 	// statsBackupIsRunning is set to 1 (true) if a backup is running.
 	statsBackupIsRunning *stats.GaugesWithMultiLabels
 
-	// statsIsInSRVKeyspace is set to 1 (true), 0 (false) whether the tablet is in the serving keyspace
-	statsIsInSRVKeyspace *stats.Gauge
+	// statsIsInSrvKeyspace is set to 1 (true), 0 (false) whether the tablet is in the serving keyspace
+	statsIsInSrvKeyspace *stats.Gauge
```

Comment:
The convention throughout the codebase `Srv` not `SRV`. I think we should change this one back.

Reply:
For more context, we talked about this in different PR recently: https://github.com/vitessio/vitess/pull/7929#discussion_r619326600

Reply:
whoops, i must have messed up basing off an old `vitessio@master` my bad! did not mean to change this

Reply:
oh @rafael we're looking at the wrong PR ... this is the cherry-pick onto our fork; that's why these variables are named differently. it's correct in the uptream PR: https://github.com/vitessio/vitess/pull/8107

Reply:
Ah lol. Sounds good.